variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/services/VariantStudySummaryService.java

```diff
@@ -67,6 +69,42 @@
         return studies.getMappedResults();
     }
 
+    public List<VariantStudySummary> findAll(int pageNumber, int pageSize) {
+        Aggregation aggregation = Aggregation.newAggregation(
+                groupAndCount(),
+                projectAndFlatten(),
+                skip(pageNumber*pageSize),
```

Comment:
This `skip` method is deprecated, please use the not deprecated one (long type as parameter)